src/components/mobile-docs-sidebar/docs.mdx

```diff
@@ -0,0 +1,709 @@
+---
+componentName: MobileDocsSidebar
+peerComponents:
+  - ProductIcon
+---
+
+<MobileDocsSidebar
+  levels={[
```

Comment:
This `levels` approach feels like a good starting point to build off of 👍  It should enable us to build up arbitrary flows as needed.